src/static/bitbucket/utils/getWidgetSettings.ts

```diff
@@ -0,0 +1,10 @@
+import { WidgetSettings } from '../../../types';
+
+export default function getWidgetSettings(): WidgetSettings {
+  return {
+    refreshInterval: 30000,
+    refreshOnlyWhenInViewport: true,
```

Comment:
I wonder if refreshing in background as a default is more expected

Reply:
Changed. FWIW "refreshing in background" only applies when the tab is in focus. We never refresh in background else.